images/kubevirt-infra-prow/publish_multiarch_prow_images.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash -x
+
+BAZEL_VERSION=3.0.0
+PROW_COMMIT=master
+repo=kubevirtci
+archs=(arm64 ppc64le)
+binaries=(clonerefs initupload entrypoint sidecar)
+
+main() {
+    local build_only
+    local tag
+    while getopts "bh" opt; do
+        case "$opt" in
+            b)
+                build_only=true
+                ;;
+            h)
+                help
+                exit 0
+                ;;
+            *)
+                echo "Invalid argument: $opt"
+                help
+                exit 1
+        esac
+    done
+    shift $((OPTIND-1))
+    tag=$(get_image_tag)
+    build_binaries
+    build_images
+    [[ $build_only ]] && return
+    push_images
+    push_manifest
+}
+
+help() {
+    cat <<EOF
+    Usage:
+        ./publish_multiarch_prow_images.sh [OPTIONS]
+
+    Build and publish mutliarch prow utility images.
```

Comment:
nit: spelling

Reply:
> nit: spelling

Should it be "Building and publishing mutliarch prow utility images"?

Reply:
No, sorry for being unclear. I meant it needs to be mu*lt*iarch instead of mu*tl*iarch :blush: